stdlib/3/posix.pyi

```diff
@@ -10,3 +10,23 @@ from typing import NamedTuple
 if sys.version_info >= (3, 3):
     uname_result = NamedTuple('uname_result', [('sysname', str), ('nodename', str),
         ('release', str), ('version', str), ('machine', str)])
+
+    times_result = NamedTuple('times_result', [
```

Comment:
Not sure why these specifically are in this file; most of the functions also live in the `posix` module. Eventually we should move most code into `posix.pyi` and thus re-export it in `os.pyi`.

Reply:
Sounds like a good idea. I was just keeping with the convention used with `uname_result`.